test/e2e/.mocharc.yml

```diff
@@ -1,7 +1,6 @@
 require:
   - 'esm'
   - 'test/mocha.env.js'
-  - 'mocha-steps'
   - 'lib/hooks'
 file:
   - 'lib/before.js'
```

Comment:
I think it would make sense to also place the `bail: true` option in the config by default, as I had forgotten to set that flag in the CLI when testing locally and was baffled why rest of the steps attempted to run.